workers/default/ojs/frontmatter.py

```diff
@@ -0,0 +1,17 @@
+from utils.ojs_api import generate_frontmatters
+
+
+def generate_frontmatter(article_id_list):
+    """
+    Generate frontmatter page for articles in OJS via Frontmatter-Plugin.
+
+    The given id_list is merged by commata.
+
+    For further details see OJS API module.
+
+    :param article_id_list: IDs of documents in OJS
+    :return: tuple containing the response code and text
+    """
+    ids = ','.join(map(str, article_id_list))
+
+    return generate_frontmatters(ids)
```

Comment:
Hmm, das ist jetzt auch irgendwie overkill. Kann man das Joinen der IDs nicht in die API ziehen? Dann machen die Funktionen nichts anderes mehr als die API-Funktionenen aufzurufen. Dann würde ich die API aus utils in `workers/default/ojs/ojs_api.py` umziehen und `frontmatter.py` und `publishing.py` löschen.